python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -1401,6 +1401,68 @@ async def is_m1m3_in_engineering_mode(self) -> bool:
 
         return m1m3_detailed_state in self.m1m3_engineering_states
 
+    async def run_m2_actuator_bump_test(
+        self,
+        actuator: int,
+        period: float,
+        force: float,
+    ) -> None:
+        """M2 actuator bump test.
+
+        Parameters
+        ----------
+        actuator : `int`
+            Id of the actuator.
+        period : `float`,
+            There will be two bumps and each bump will wait for (2 * period)
+            seconds.
+        force : `float`, optional
```

Comment:
Is this really optional? If so, what is the default value?

Reply:
removed the optional